scipy/linalg/decomp.py

```diff
@@ -1267,3 +1267,138 @@ def hessenberg(a, calc_q=False, overwrite_a=False, check_finite=True):
     q, info = orghr(a=hq, tau=tau, lo=lo, hi=hi, lwork=lwork, overwrite_a=1)
     _check_info(info, 'orghr (hessenberg)', positive=False)
     return h, q
+
+
+def cdf2rdf(w, v):
+    """
+    Converts complex eigensystems to real eigenvalues in a block diagonal form
+    and the associated real eigenvectors.
+
+    If the eigensystem:
+
+    `w, v = scipy.linalg.eig(X)`
+    or
+    `w, v = numpy.linalg.eig(X)`
```

Comment:
We don't need numpy references here.

Reply:
I think it would be appropriate to use both scipy and numpy references here, because cdf2rdf takes both scipy.linalg.eig results (where v : single square matrix -> 2D array), and numpy.linalg.eig results (where v : stacked square matrices -> 3D array) ?

Reply:
You can certainly mention it in the docstring but when printed in the terminal this is a bit of a waste of the precious vertical space.